### DISCUSSION TITLE
Short example where to put own data?

### DISCUSSION THREAD

**Discussion by MSDNAndi** [lifeiteng/valle#107] [General]
VALL-E works with very short speech samples.
I am staring at the repo and do not find the key point.
Where to put your own voice voice sample.

If somebody has a "minimal example" to follow, I would appreciate it.

**Comment by maxgoff** [comment]
Following the instructions, I am successfully getting training done now (examples/libritts).  Up to Epoch 4 at the moment.   I see in that directory a subdirectory ./prompts which contains 2 sets of .txt /.wav files.  I am wondering if I substitute those specific files with my own files if that will result in my own voice samples being used to train.  Thoughts?

**Comment by lifeiteng** [comment]
you can prepare your own dataset by using `lhoste` more info in https://github.com/lifeiteng/vall-e/issues/87, `prompts` are used in Inference not Training.

**Reply by maxgoff** [reply]
If I use my own voice as prompts during inference, will the output reflect the characteristics of my voice in the prompt files?
To personalize the output voice, does it require a full training set with my voice such as are provided for training purposes, or will the vocal characteristics from the prompt during inference be used?  I can see where prompting may be for providing new text for processing.  But if that's the case, then I don't see why the .wav files would be needed.  In any event, thank you for this great work you are doing!

**Reply by chenjiasheng** [reply]
> If I use my own voice as prompts during inference, will the output reflect the characteristics of my voice in the prompt files?


Yes.

> To personalize the output voice, does it require a full training set with my voice such as are provided for training purposes, or will the vocal characteristics from the prompt during inference be used?

Training on a huge voice dataset including your own voice, or fine-tune the pretrained model with your own voice, can achieve better naturalness and similarity for your own voice. 

> But if that's the case, then I don't see why the .wav files would be needed.

The prompt .wav as well as the transcrpit is needed for inference, no matter whether you have trained or fine-tuned the model with your own voice.
